Assets/Scripts/Kinematic.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Kinematic serves as a custom data type for storing the movement and rotation of the agent
+/// </summary>
+public class Kinematic
+{
+    //Instantaneous velocity
```

Comment:
Instead of writing // Section Title, section them by 

`[Header("Section Title")]`

which adds a useful header within the unity inspector!

Reply:
Sounds good!